backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  createdAt                     DateTime                    @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                     DateTime                    @updatedAt @map("updated_at") @db.Timestamp(6)
+  type                          ApplicationReviewStatusEnum
+  label                         String?
+  externalReference             String?                     @map("external_reference")
+  acceptsPostmarkedApplications Boolean?                    @map("accepts_postmarked_applications")
+  phoneNumber                   String?                     @map("phone_number")
+  listingId                     String?                     @map("listing_id") @db.Uuid
+  listings                      listings?                   @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  paperApplications             PaperApplications[]
+
+  @@map("application_methods")
+}
+
+// Note: [additional_phone_number, additional_phone_number_type, household_size] formerly max length 16; 
+//       [contact_preferences, income] formerly max length 64;
+//       [app_url] formerly max length 256; 
+// Note: [contact_preferences] formerly max array length of 8
+// Note: [household_member] formerly had max array lenght of 32
+// Note: missing virtual [flagged] field
+model Applications {
+  id                           String                        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                    DateTime                      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                    DateTime                      @updatedAt @map("updated_at") @db.Timestamp(6)
+  deletedAt                    DateTime?                     @map("deleted_at") @db.Timestamp(6)
+  appUrl                       String?                       @map("app_url")
+  additionalPhone              Boolean?                      @map("additional_phone")
+  additionalPhoneNumber        String?                       @map("additional_phone_number")
+  additionalPhoneNumberType    String?                       @map("additional_phone_number_type")
+  contactPreferences           String[]                      @map("contact_preferences")
+  householdSize                Int?                          @map("household_size")
+  housingStatus                String?                       @map("housing_status")
+  sendMailToMailingAddress     Boolean?                      @map("send_mail_to_mailing_address")
+  householdExpectingChanges    Boolean?                      @map("household_expecting_changes")
+  householdStudent             Boolean?                      @map("household_student")
+  incomeVouchers               Boolean?                      @map("income_vouchers")
+  income                       String?
+  incomePeriod                 IncomePeriodEnum              @map("income_period")
+  preferences                  Json
```

Comment:
I know it wasn't before, but it seems like this should be optional (and recognizing a multiselection question refactor is out of scope) - did we lose typing here for both programs and preferences?

Reply:
typing for these would occur at a higher level, I don't know if we really need it to be optional since technical an empty `{}` or `[]` would work just as well 

I feel like that would be a good thing for us to consider later though (as a sort of pass through the schema)

Reply:
Wondering if this and programs beneath it should be the same, but either works for me!